README.md

```diff
@@ -0,0 +1,4 @@
+This contribution is no longer maintained, this formalization is now part of the GeoCoq library.
+
+https://github.com/GeoCoq/GeoCoq
```

Comment:
```suggestion
This formalization is now maintained as part of the GeoCoq library: https://github.com/GeoCoq/GeoCoq
```
@jnarboux everything in this organization is unmainained. I agree that a pointer to GeoCoq is good though.